dp-bulletin-api.nomad

```diff
@@ -0,0 +1,141 @@
+job "dp-bulletin-api" {
+  datacenters = ["eu-west-1"]
+  region      = "eu"
+  type        = "service"
+
+  update {
+    stagger          = "60s"
+    min_healthy_time = "30s"
+    healthy_deadline = "2m"
+    max_parallel     = 1
+    auto_revert      = true
+  }
+
+  group "web" {
+    count = "{{WEB_TASK_COUNT}}"
+
+    constraint {
+      attribute = "${node.class}"
+      value     = "web"
+    }
+
+    restart {
+      attempts = 3
+      delay    = "15s"
+      interval = "1m"
+      mode     = "delay"
+    }
+
+    task "dp-bulletin-ap-web" {
+      driver = "docker"
+
+      artifact {
+        source = "s3::https://s3-eu-west-1.amazonaws.com/{{DEPLOYMENT_BUCKET}}/dp-bulletin-api/{{REVISION}}.tar.gz"
```

Comment:
This is the old url pattern.  You will still see this around because concourse is able to handle both, but it is technically deprecated.  The following is the new pattern:
```suggestion
        source = "s3::https://s3-eu-west-1.amazonaws.com/{{DEPLOYMENT_BUCKET}}/dp-bulletin-api/{{PROFILE}}/{{RELEASE}}.tar.gz"
```